@ecomplus/storefront-app/src/components/html/CreditCardForm.html

```diff
@@ -3,13 +3,18 @@
   @submit.prevent="submit"
   novalidate
 >
-  <div
-    v-show="isWaitingCardHash" 
-    class="spinner-border"
-    role="status"
-  >
-    <span class="sr-only">Loading...</span>
-  </div>
+  <transition name="fade">
+    <div
+      v-show="isWaitingCardHash" 
+      class="credit-card-loading">
```

Comment:
```suggestion
      class="credit-card__loading"
    >
```

Já falamos sobre [BEM](http://getbem.com/introduction/) né...